tests/Browser/ResultsTest.php

```diff
@@ -157,8 +151,8 @@ public function test_apply_changes_button_submits_new_review_status()
             $browser->loginAs(User::factory()->create())
                 ->visit(new ResultsPage($mismatch->item_id))
                 ->within($dropdownComponent, function ($dropdown) {
-                    // make sure first value is displayed as it should
-                    $dropdown->assertOption('Mismatch on Wikidata')
+                    // make sure the item's initial review_status is 'pending'
+                    $dropdown->assertVue('value', null)
```

Comment:
Nice! I forgot, that `pending` wasn't part of the `ReviewDecision` enum :scream: